src/resources/js/utils/tec-copy-to-clipboard.js

```diff
@@ -0,0 +1,47 @@
+tribe.copyToClipboard = tribe.copyToClipboard || {};
+
+( function ( $, obj ) {
+	'use strict';
+
+	obj.selectors = {
+		copyButton: '.tribe-copy-to-clipboard',
+	};
+
+	/**
+	 * Initialize system info opt in copy
+	 */
+	obj.setupCopyButton = function () {
+		const clipboard = new ClipboardJS( obj.selectors.copyButton );
+		const notice     = $( $( obj.selectors.copyButton ).data( 'notice-target' ) );
+
+		//Prevent Button From Doing Anything Else
+		$( obj.selectors.copyButton ).on(
+			'click',
+			function ( e ) {
+				e.preventDefault();
+			}
+		);
```

Comment:
Bind to the document so this will work with any copyButton added at run time.

```suggestion
		$( document ).on(
			'click',
			obj.selectors.copyButton,
			function ( e ) {
				e.preventDefault();
			}
		);
```